### PR TITLE
[FrameworkBundle][Validator] Move Validator passes to the component

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by chalasr** [comment]
appveyor failure unrelated

File: src/Symfony/Component/Validator/DependencyInjection/AddConstraintValidatorsPass.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+namespace Symfony\Component\Validator\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Argument\ServiceClosureArgument;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+class AddConstraintValidatorsPass implements CompilerPassInterface
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/DependencyInjection/AddConstraintValidatorsPass.php]: missing author

File: src/Symfony/Component/Validator/DependencyInjection/AddValidatorInitializersPass.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+class AddValidatorInitializersPass implements CompilerPassInterface
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/DependencyInjection/AddValidatorInitializersPass.php]: same here

Comment (chalasr) [inline src/Symfony/Component/Validator/DependencyInjection/AddValidatorInitializersPass.php]: fixed

**Comment by chalasr** [comment]
rebased

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**